CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## Next
 
+### Docs
+* Button en Button row: Documentatie aangepast en nieuwe varianten gemaakt voor de Button-row om de varianten van de Buttons te demonstreren. ([#1898](https://github.com/dso-toolkit/dso-toolkit/issues/1898)) 
```

Comment:
Sinds kort linten we de commit messages en de CHANGELOG, zie de `dangerfile.ts` voor de daadwerkelijke implementatie en de documentatie: https://dso-toolkit.nl/58.0.1/voor-maintainers/change-management-notatie

Het eerste stukje is wat we de "scope" noemen. De scope "Schrijven We Altijd Zo", maar in jouw geval is dat 'n beetje lastig.. Want "Button En Button Row" is nogal suf. Ik denk dat we wel wegkomen met "Button & Button Row", kan je dat eens proberen?

Reply:
👍